crates/iceberg/src/arrow/record_batch_transformer.rs

```diff
@@ -429,6 +471,27 @@ impl RecordBatchTransformer {
                 let vals: Vec<Option<f64>> = vec![None; num_rows];
                 Arc::new(Float64Array::from(vals))
             }
+            (DataType::RunEndEncoded(_, _), Some(PrimitiveLiteral::String(value))) => {
```

Comment:
Should we in general encode constant columns as REE? Or should we make this custom per field? For the file path it definitely makes sense to run-end encode.